org/mozilla/jss/pkcs11/PK11KeyWrapper.c

```diff
@@ -17,6 +17,8 @@
 #include <jssutil.h>
 #include <pk11util.h>
 #include <Algorithm.h>
+#include "StaticVoidPointer.h"
+#include "jssconfig.h"
```

Comment:
@jmagne -- sorry, I forgot one line, that's why it was failing. :-) 

`jssconfig.h` defines `HAVE_NSS_OAEP` from the build system. Now it works again.